tests/TestKernel.php

```diff
@@ -53,17 +55,52 @@ function (ContainerBuilder $container) {
                     'symfonycasts.messenger_monitor.storage.doctrine_connection'
                 )->setPublic(true);
 
+                $container->setAlias('test.messenger.bus.default', 'messenger.bus.default')->setPublic(true);
+                $container->setAlias('test.messenger.receiver_locator', 'messenger.receiver_locator')->setPublic(true);
+
+                $messageHandlerDefinition = new Definition(TestableMessageHandler::class);
+                $messageHandlerDefinition->addTag('messenger.message_handler');
+                $container->setDefinition('test.message_handler', $messageHandlerDefinition);
+
                 $container->setParameter('kernel.secret', 123);
                 $container->prependExtensionConfig(
```

Comment:
I don't think you need perpend - I think you just need `loadFromExtension()` - and down below for Doctrine too.